target/linux/ath79/generic/base-files/etc/hotplug.d/firmware/10-ath9k-eeprom

```diff
@@ -38,6 +38,7 @@ case "$FIRMWARE" in
 		caldata_extract "art" 0x1000 0x440
 		ath9k_patch_mac $(macaddr_add $(mtd_get_mac_ascii u-boot-env "athaddr") +1)
 		;;
+	engenius,emr3000|\
```

Comment:
You are using the same calibration data twice?!
Please disentangle the use and implementation of both radios for us ...

Reply:
caldata_extract "art" 0x1000 0x440 for 2.4G
caldata_extract "art" 0x5000 0x844 for 5G
I don't usign the same calibration data twice.
If there are any errors in the scripts?
The WIFI works with 2.4g and 5g radio.

Reply:
You are using mtd-cal-data in DTS as well. Only one of them can be right ...

Reply:
Thanks for comment.
Remove mtd-cal-data from DTS.